raft/src/main/java/org/apache/kafka/raft/RequestManager.java

```diff
@@ -372,12 +372,12 @@ void onRequestSent(long correlationId, long timeMs) {
         @Override
         public String toString() {
             return String.format(
-                "ConnectionState(node=%s, state=%s, lastSendTimeMs=%d, lastFailTimeMs=%d, inFlightCorrelationId=%d)",
+                "ConnectionState(node=%s, state=%s, lastSendTimeMs=%d, lastFailTimeMs=%d, inFlightCorrelationId=%s)",
                 node,
                 state,
                 lastSendTimeMs,
                 lastFailTimeMs,
-                inFlightCorrelationId
+                inFlightCorrelationId.isPresent() ? inFlightCorrelationId.getAsLong() : null
```

Comment:
how about using `undefined` to replace "null"?

Reply:
You are right, this makes it easier to understand
it has been modified.